contracts/modules/legacy/LegacyPooledStaking.sol

```diff
@@ -1288,10 +1291,14 @@ contract LegacyPooledStaking is IPooledStaking, MasterAwareV2 {
         trancheDeposit,
         firstTrancheId + i,
         tokenId,
-        migrationData.managerAddress
+        address(this)
       );
     }
 
+    if (tokenId != 0) {
```

Comment:
some comment would be good here to say tokenId remains 0 when no deposits are made (fundation wallet)